README.md

```diff
@@ -1,45 +1,51 @@
 # RAML Mock Server 
 
-##### Brief introduction  -  [RAML](https://raml.org/)
-
-RAML (RESTful API Modeling Language) provides a structured, unambiguous format for describing a RESTful API. 
-It allows you to describe your API; the endpoints, the HTTP methods to be used for each one, any parameters and 
-their format, what you can expect by way of a response and more.
-
+##### Brief introduction   
 
+This application allows you to run a REST API mock sever which uses [RAML](https://raml.org/). 
 
 # Demo
-The only file you need to run your API demo is **demo/raml/api.raml**, which is used by the mock server to create endpoints.
+The only file you need to run your API demo is **raml/api.raml**, which is used by the mock server to create endpoints.
 
 api.raml contains one basic example of **GET** request and to explore more please check the RAML 
 [developers page](https://raml.org/developer).
 
 
 #### Execute below command to start the demo:
```

Comment:
Looks like the level of the heading is too deep.

Reply:
Done, using bullet points